operations/README.md

```diff
@@ -29,84 +32,56 @@ az login
 
 - Navigate to the provided login URL and input the provided token
 
-This only needs to be on first run and after your Azure credentials expire. The state of your login will be persisted in a Docker volume.
-
-
-## Set the default Azure subscription (on first run only)
-
-If you have access to multiple Azure subscriptions, ensure you set the intended subscription as the default, or you will see errors from Terraform:
-
-```
-az account list
-az account set -s {subscription-id}
-```
+This only needs to be on first run and after your Azure credentials expire. The state of your login will be persisted in a Docker volume that is shared across stages.
 
 
 ## Using Terraform
 
-Initialize the Terraform environment (this needs to completed on first run and after every new module)
+A typical Terraform deployment may look as follows:
 
-```
-terraform init
-```
+```aidl
+make TF_ENV=x tf-01-network
```

Comment:
see above about aidl & providing example for values of `x`. Especially since this is the canonical example.